### DISCUSSION TITLE
[Feature Request] Allow exporting character names and IDs as a list

### DISCUSSION THREAD

**Discussion by NickeManarin** [character-map-uwp/Character-Map-UWP#222] [Ideas]
It would be nice to be able to export font characters name and IDs as a list. 
For example, I want to generate a C# enum based on all glyphs of a custom symbol font.

**Comment by JohnnyWestlake** [comment]
@NickeManarin Not a timely reply, but if you can provide a small bit of pseudo code of what you'd want the output to look like I can probably put this in
